packages/open-payments/src/client/index.ts

```diff
@@ -118,9 +121,36 @@ const parseKey = (
   }
 }
 
-const createDeps = async (
-  args: Partial<CreateAuthenticatedClientArgs>
-): Promise<ClientDeps> => {
+const createUnauthenticatedDeps = async ({
```

Comment:
Thought it would be okay to separate the unauthenticated dependencies from the authenticated ones. This way, when initializing an unauthenticated client, the authentication server OpenAPI schema will not be loaded.